subjects/devops/clean_the_list/README.md

```diff
@@ -0,0 +1,32 @@
+## clean_the_list
+
+### Instructions
+
+Create a file `shopping.py` that contains a function `clean_list` which takes in a list of strings and performs the following operations on each list item:
+
+- Removes all spaces before and after (but not between words).
+- Capitalizes the first letter (first letter only, other ones should be to lowercase).
+- Adds its index number before a separator `x/ `.
+- An empty list as input should return an empty list as output.
+- And don't forget the milk !!!! (add it at the end of the list if missing).
```

Comment:
usually `!` is attached at the world it is following. Also I do like the joke but maybe only one `!` will be enough.

I suggest it to became `... milk! (add ..`

Reply:
Thanks I changed it.